eval.py

```diff
@@ -67,6 +67,9 @@ def eval(**args):
     if args['load_type'] == 'train_val':
         eval_loader = loader['valid']
 
+    elif args['load_type'] == 'train':
```

Comment:
Are we now allowing training to be loaded in `eval.py` as well? If so, we just need to change the sys.exit error.